dolphinscheduler-master/src/test/java/org/apache/dolphinscheduler/server/master/consumer/TaskPriorityQueueConsumerTest.java

```diff
@@ -37,18 +37,14 @@
 import java.util.Date;
 import java.util.concurrent.TimeUnit;
 
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.*;
```

Comment:
We are supposed to avoid wildcard `*` imports. Usually spotless will detect and block this. I'm not sue why it didn't work here. I will take a look at it.

Reply:
> We are supposed to avoid wildcard `*` imports. Usually `spotless` detects and blocks this. I'm not sure why it didn't work here. I will take a look at it.

done